testutils/rand/rand.go

```diff
@@ -32,6 +38,21 @@ func I64Between(lower int64, upper int64) int64 {
 	return rand.Int63n(upper-lower) + lower
 }
 
+// NormalizeString normalizes a string as NFKC
+func NormalizeString(str string) string {
+	return norm.NFKC.String(str)
+}
```

Comment:
we already have a function for this in the utils package

Reply:
There's a circular import with `utils` package :/

Reply:
bummer